Command/RequestAPICommand.php

```diff
@@ -14,24 +15,32 @@ protected function configure()
     {
         $this
             ->setName('ac:request-api')
-            ->addArgument('client-key', InputArgument::REQUIRED)
-            ->addArgument('rest-url', InputArgument::REQUIRED)
-            ->setDescription('Request REST end-points');
+            ->addArgument('rest-url', InputArgument::REQUIRED, "REST api endpoint, like /rest/api/2/issue/{issueIdOrKey}")
+            ->addOption('client-key', "c", InputOption::VALUE_OPTIONAL | InputOption::VALUE_REQUIRED, "Client-key from tenant")
```

Comment:
`InputOption::VALUE_OPTIONAL | InputOption::VALUE_REQUIRED` — this operation would always resolve into `InputOption::VALUE_OPTIONAL`, correct? Could you please explain the logic of comparison? 🤔